### DISCUSSION TITLE
Hard Rock Mania Functionality

### DISCUSSION THREAD

**Discussion by tailsdk** [ppy/osu#26584] [Q&A]
Hard Rock currently functions a little bit differently between lazer and stable.

In stable there is a 1.4 times scaling that decreases the size of all hit windows after OD meaning also the perfect 300 hit windows get even smaller.

On lazer the 1.4 times scaling is applied before and so the OD just increases instead capped out at 10.

The reason I am asking is because there are updsides and downsides to both.

In stable Hard Rock cannot be rankable because it can be abused a lot because of the small hit windows letting you play jack maps at really high BPM without having to click the correct buttons but just mashing all buttons. Some people likes to use it as an accuracy challenge mod to improve their accuracy because the hit windows become so small.

In lazer currently the cheese on jack maps isnt more present than it already is with no mod since it just increases OD, but it does remove some of the accuracy challenge part from the mod thst it was originally used for on stable.

My question would be what is prefered.

**Comment by deltaloni** [comment]
would prefer stable's version due to accuracy training. having to switch between the two clients makes it hard to train accuracy. hopefully we get something in lazer that's similar and can help us adjust timing windows on mania
